libs/application/templates/driving-license/src/forms/application.ts

```diff
@@ -141,7 +141,7 @@ Veitir ökuréttindi til að stjórna bifhjóli:
             buildKeyValueField({
               label: 'Umsækjandi',
               value: ({ externalData: { nationalRegistry } }) =>
-                nationalRegistry.data.fullName,
+                (nationalRegistry.data as { fullName: string }).fullName,
```

Comment:
Should we create a ts interface for the nationalRegistry data instead of doing this?